Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Matomo/CommunityPlugin GroupPermissions](https://hosted.weblate.org/projects/matomo/communityplugin-grouppermissions/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/matomo/-/communityplugin-grouppermissions/horizontal-auto.svg)

Chardonneaur commented:
@MichaelRoosz just to let you know that once the new update released, I will shot another video explaining how to translate a community plugin and have it live for Matomo. It is part of a huge process to train end users about the concept of what is Matomo and how they can contribute to it.
Thank you so much for creating this plugin which took me only 10 minutes to translate from English to French. :)

weblate committed:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (24 of 24 strings)

Translation: Matomo/CommunityPlugin GroupPermissions
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-grouppermissions/pt/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Mayara da Silva Oliveira <mayara.s.oliveira@edu.ufes.br>

weblate committed:
Added translation using Weblate (Malay)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (24 of 24 strings)

Translation: Matomo/CommunityPlugin GroupPermissions
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-grouppermissions/es/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: gallegonovato <fran-carro@hotmail.es>

weblate committed:
Translated using Weblate (Basque)

Currently translated at 100.0% (24 of 24 strings)

Translation: Matomo/CommunityPlugin GroupPermissions
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-grouppermissions/eu/

[ci skip]

Translated using Weblate (Basque)

Currently translated at 95.8% (23 of 24 strings)

Translation: Matomo/CommunityPlugin GroupPermissions
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-grouppermissions/eu/

[ci skip]

Co-authored-by: Asier Iturralde Sarasola <asier.iturralde@aldatsa.eus>
Co-authored-by: Hosted Weblate <hosted@weblate.org>

weblate committed:
Added translation using Weblate (Welsh)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Translated using Weblate (Swedish)

Currently translated at 100.0% (24 of 24 strings)

Translation: Matomo/CommunityPlugin GroupPermissions
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-grouppermissions/sv/

[ci skip]

Translated using Weblate (Swedish)

Currently translated at 79.1% (19 of 24 strings)

Translation: Matomo/CommunityPlugin GroupPermissions
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-grouppermissions/sv/

[ci skip]

Translated using Weblate (Swedish)

Currently translated at 41.6% (10 of 24 strings)

Translation: Matomo/CommunityPlugin GroupPermissions
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-grouppermissions/sv/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Marcus Österberg <m@tba.nu>

weblate committed:
Translated using Weblate (French)

Currently translated at 100.0% (24 of 24 strings)

Translation: Matomo/CommunityPlugin GroupPermissions
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-grouppermissions/fr/

[ci skip]

Translated using Weblate (French)

Currently translated at 100.0% (24 of 24 strings)

Translation: Matomo/CommunityPlugin GroupPermissions
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-grouppermissions/fr/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: J. Lavoie <j.lavoie@net-c.ca>
Co-authored-by: Ronan Chardonneau <contact@ronan-chardonneau.fr>

weblate committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (24 of 24 strings)

Translation: Matomo/CommunityPlugin GroupPermissions
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-grouppermissions/pt_BR/

[ci skip]

Translated using Weblate (Portuguese (Brazil))

Currently translated at 62.5% (15 of 24 strings)

Translation: Matomo/CommunityPlugin GroupPermissions
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-grouppermissions/pt_BR/

[ci skip]

Co-authored-by: Claudio Filho F Filho <filhocf@gmail.com>
Co-authored-by: César Bridi <cbridi@gmail.com>
Co-authored-by: Hosted Weblate <hosted@weblate.org>

weblate committed:
Translated using Weblate (Turkish)

Currently translated at 100.0% (24 of 24 strings)

Translation: Matomo/CommunityPlugin GroupPermissions
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-grouppermissions/tr/

[ci skip]

Translated using Weblate (Turkish)

Currently translated at 100.0% (24 of 24 strings)

Translation: Matomo/CommunityPlugin GroupPermissions
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-grouppermissions/tr/

[ci skip]

Translated using Weblate (Turkish)

Currently translated at 100.0% (24 of 24 strings)

Translation: Matomo/CommunityPlugin GroupPermissions
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-grouppermissions/tr/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Kaya Zeren <kayazeren@gmail.com>
Co-authored-by: Oğuz Ersen <oguz@ersen.moe>

weblate committed:
Added translation using Weblate (Azerbaijani)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Luxembourgish)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Translated using Weblate (Albanian)

Currently translated at 100.0% (24 of 24 strings)

Translation: Matomo/CommunityPlugin GroupPermissions
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-grouppermissions/sq/

[ci skip]

Co-authored-by: Besnik Bleta <besnik@programeshqip.org>
Co-authored-by: Hosted Weblate <hosted@weblate.org>

weblate committed:
Translated using Weblate (Indonesian)

Currently translated at 100.0% (24 of 24 strings)

Translation: Matomo/CommunityPlugin GroupPermissions
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-grouppermissions/id/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Taufik Adi Wicaksono <taufikadi.wicaksono@tutamail.com>

weblate committed:
Added translation using Weblate (Esperanto)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (24 of 24 strings)

Translation: Matomo/CommunityPlugin GroupPermissions
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-grouppermissions/ca/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: raf <rafroset@gmail.com>

weblate committed:
Translated using Weblate (Ukrainian)

Currently translated at 4.1% (1 of 24 strings)

Translation: Matomo/CommunityPlugin GroupPermissions
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-grouppermissions/uk/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Ihor Hordiichuk <igor_ck@outlook.com>

weblate committed:
Translated using Weblate (Gujarati)

Currently translated at 100.0% (24 of 24 strings)

Translation: Matomo/CommunityPlugin GroupPermissions
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-grouppermissions/gu/

[ci skip]

Added translation using Weblate (Gujarati)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Kalpesh Mahida <kalpesh.mahida@gmail.com>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Translated using Weblate (Bulgarian)

Currently translated at 100.0% (24 of 24 strings)

Translation: Matomo/CommunityPlugin GroupPermissions
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-grouppermissions/bg/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Ministry of Electronic Governance - Bulgaria <dvdimov@e-gov.bg>

weblate committed:
Translated using Weblate (Italian)

Currently translated at 54.1% (13 of 24 strings)

Translation: Matomo/CommunityPlugin GroupPermissions
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-grouppermissions/it/

[ci skip]

Co-authored-by: Bertani Leonardo <bertani.leonardo03@gmail.com>
Co-authored-by: Hosted Weblate <hosted@weblate.org>

weblate committed:
Translated using Weblate (Japanese)

Currently translated at 100.0% (24 of 24 strings)

Translation: Matomo/CommunityPlugin GroupPermissions
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-grouppermissions/ja/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Kozue Ono <kozue_ono@matomo.jp>

weblate committed:
Added translation using Weblate (Armenian)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Translated using Weblate (Greek)

Currently translated at 100.0% (24 of 24 strings)

Translation: Matomo/CommunityPlugin GroupPermissions
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-grouppermissions/el/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Vasilis Lourdas <dev@lourdas.eu>

weblate committed:
Added translation using Weblate (Bengali)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Translated using Weblate (Dutch)

Currently translated at 100.0% (24 of 24 strings)

Translation: Matomo/CommunityPlugin GroupPermissions
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-grouppermissions/nl/

[ci skip]

Translated using Weblate (Dutch)

Currently translated at 20.8% (5 of 24 strings)

Translation: Matomo/CommunityPlugin GroupPermissions
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-grouppermissions/nl/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Ingo Van den Broeck <in.vandenbroeck@vmm.be>

weblate committed:
Translated using Weblate (German)

Currently translated at 100.0% (24 of 24 strings)

Translation: Matomo/CommunityPlugin GroupPermissions
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-grouppermissions/de/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: J. Lavoie <j.lavoie@net-c.ca>

weblate committed:
Added translation using Weblate (Kurdish)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Translated using Weblate (Chinese (Simplified))

Currently translated at 100.0% (24 of 24 strings)

Translation: Matomo/CommunityPlugin GroupPermissions
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-grouppermissions/zh_Hans/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: YYT <1209897320@qq.com>

weblate committed:
Added translation using Weblate (Dhivehi)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Sinhala)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Tamazight (Central Atlas))

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Translated using Weblate (Norwegian Bokmål)

Currently translated at 79.1% (19 of 24 strings)

Translation: Matomo/CommunityPlugin GroupPermissions
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-grouppermissions/nb_NO/

[ci skip]

Co-authored-by: Allan Nordhøy <epost@anotheragency.no>
Co-authored-by: Hosted Weblate <hosted@weblate.org>

weblate committed:
Translated using Weblate (Chinese (Traditional))

Currently translated at 8.3% (2 of 24 strings)

Translation: Matomo/CommunityPlugin GroupPermissions
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-grouppermissions/zh_Hant/

[ci skip]

Co-authored-by: HO Gin Wang <hoginwang1996@gmail.com>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
